syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.examples.helloworld";
option java_outer_classname = "HelloWorldProto";
option objc_class_prefix = "HLW";

package test;

service Test {
    // Sends a greeting
    rpc TestOpen (OpenSend) returns (OpenReply) {}
  }
  
  // The request message containing the user's name.
  message OpenSend {
    bytes inarg = 1;
    uint32 outlen = 2;
  }
  
  // The response message containing the greetings
  message OpenReply {
    bytes outarg = 1;
  }
  
